bee-message/src/output/foundry.rs

```diff
@@ -366,13 +367,26 @@ impl StateTransitionVerifier for FoundryOutput {
             return Err(StateTransitionError::MissingAliasForFoundry);
         }
 
+        let token_id = next_state.token_id();
+        let output_tokens = context.output_native_tokens.get(&token_id).copied().unwrap_or_default();
```

Comment:
Would the default always lead to a `StateTransitionError::InconsistenNativeTokens` in line 379?

Reply:
No, why ?

Reply:
Just trying to understand to logic. I don't see the exact types unfortunately but I guess the default would be a `0u64` or something. Line 379 would mean for the default that no tokens are minted at all during the creation which seemed strange to me, but I am probably having a misunderstanding here.

Reply:
You can create the foundry and mint tokens at a later stage if you want to

Reply:
alright, thanks for the explanation.